### PR TITLE
Update studio-tasks-git.md

### PR DESCRIPTION
**Pull request by nanda-nainadurai**
Updated text to use personal access token

*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: doc_source/studio-tasks-git.md [hunk 1]
```diff
@@ -16,12 +16,12 @@ Amazon SageMaker Studio can connect only to a local repository\. In this example
 
 1. Choose **CLONE**\.
 
-1. If the repo requires credentials, you are prompted to enter your username and password\.
+1. If the repo requires credentials, you are prompted to enter your username and password/personal access token. `From Aug 13, 2021 password does not work and you need to use a personal access token.`
```
Comment (IvyBazan) [inline doc_source/studio-tasks-git.md:19]: Suggested edit:
"If the repo requires credentials, you are prompted to enter your username and personal access token."

**Review by nanda-nainadurai — COMMENTED** [review]
Suggested edit accepted. Thanks!

File: doc_source/studio-tasks-git.md (hunk 1 above)
Comment (nanda-nainadurai) [reply doc_source/studio-tasks-git.md]: Done.

**Review by IvyBazan — APPROVED** [review]
LGTM

**Pull request merged**